products/developments/version.env

```diff
@@ -24,6 +24,8 @@ HPD_HISTOICAL_VERSION=20230630
 DOB_DATA_DATE=20230718
 DOB_NOW_APPS_VERSION=20230710
 DOB_NOW_PERMITS_VERSION=20230720
-DOB_COFOS_VERSION=20230719
+DOB_COFOS_VERSION=20230709
 DOITT_BUILDINGS_VERSION=20230702
-DOITT_BUILDINGS_HISTORICAL_VERSION=20230213
\ No newline at end of file
+DOITT_BUILDINGS_HISTORICAL_VERSION=20230213
+
+OTHER_DOB_SOURCES=20230827
```

Comment:
looks like `DOB_DATA_DATE` used to do the job of `OTHER_DOB_SOURCES`. maybe good to keep either the new or the old envar?

Reply:
Gonna rework all of this and use a recipe.yml I think, but for now this is to re-generate with exact same sources as original 23Q2. I should at least rename this probably - it's for some other DOB sources which were unpinned before

Reply:
Oh I see - looked at the code and you're right, DOB_DATA_DATE is currently not being used at all. I'll fix